File: 22-feedback/python-sdk/feedback.py
import os, requests
API = "https://cost-katana-backend.store/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def submit_feedback(request_id, rating):
    res = requests.post(f"{API}/v1/feedback",
        json={"requestId": request_id, "rating": rating},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print("✅ Feedback submitted")
    return res.json()["data"]

File: 22-feedback/npm-package/feedback.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api';
const KEY = process.env.COST_KATANA_API_KEY;

export async function submitFeedback(requestId: string, rating: number) {
  const res = await axios.post(`${API}/request-feedback/${requestId}/feedback`, { rating },
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Feedback submitted');
  return res.data.data;
}
